src/main/java/org/opensearch/timeseries/model/Entity.java

```diff
@@ -9,7 +9,7 @@
  * GitHub history for details.
  */
 
-package org.opensearch.ad.model;
+package org.opensearch.timeseries.model;
```

Comment:
nit: we use detectorId in some method input name which should change now to configID or timeSeriesID. Also some mentions of detector in comments (these comments though also refer to bwc between versions that are only really relatable to AD and are helpful as they are though so I don't mind to either keep as is (e.g. getModelId) and maybe just mentioning nuance when applied to new forecaster. Can also be dealt with when we get to the point forecasting code is calling these methods.

Reply:
I fixed comments and parameters of getModelId. I also modified getModelId to not consider bwc when this is used in forecasting.